app/src/main/res/layout/fragment_habit_view.xml

```diff
@@ -9,180 +9,193 @@
-        android:layout_width="359dp"
-        android:layout_height="171dp"
-        android:layout_marginTop="60dp"
+        android:layout_width="360dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="64dp"
         android:background="@drawable/text_view_background"
         android:ems="10"
-        android:text="@string/descriptionTextBoxLabel"
         android:textSize="16sp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/public_button" />
+        app:layout_constraintTop_toTopOf="parent" />
 
     <TextView
         android:id="@+id/dateHabitTextBox"
-        android:layout_width="359dp"
-        android:layout_height="48dp"
+        android:layout_width="360dp"
+        android:layout_height="wrap_content"
         android:layout_marginTop="32dp"
         android:background="@drawable/text_view_background"
         android:ems="10"
-        android:text="@string/dateTextBoxLabel"
         android:textSize="16sp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.5"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/descriptionHabitTextBox" />
 
-    <TextView
-        android:id="@+id/ratingHabitTextBox"
-        android:layout_width="359dp"
-        android:layout_height="48dp"
-        android:layout_marginTop="32dp"
-        android:background="@drawable/text_view_background"
-        android:ems="10"
-        android:text="@string/ratingTextBoxLabel"
-        android:textSize="16sp"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.5"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/dateHabitTextBox" />
-
     <androidx.appcompat.widget.AppCompatButton
         android:id="@+id/eventHistoryButton"
-        android:layout_width="224dp"
-        android:layout_height="58dp"
-        android:layout_marginTop="32dp"
+        android:layout_width="180dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="64dp"
         android:background="@drawable/text_button_background"
-        android:text="@string/historyButtonLabel"
+        android:text="@string/event_history"
         android:textColor="@color/white"
-        android:textSize="24sp"
+        android:textSize="18sp"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.5"
+        app:layout_constraintHorizontal_bias="0.497"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/ratingHabitTextBox" />
+        app:layout_constraintTop_toBottomOf="@+id/public_button" />
 
     <CheckBox
-        android:id="@+id/monday_button"
+        android:id="@+id/thursday_button"
```

Comment:
Not sure why the diff thinks all the `id`s are changed. All days of the week buttons are still here, might be in a different order that's all.